ansible/modules/hashivault/hashivault_secret_engine.py

```diff
@@ -81,6 +120,10 @@ def hashivault_secret_engine(module):
     backend = params.get('backend')
     description = params.get('description')
     config = params.get('config')
+    if 'default_lease_ttl' in config:
+        config['default_lease_ttl'] = int(config['default_lease_ttl'])
```

Comment:
My concern here is this looks like it isn't going to be backward compatible. Anyone with a default_lease_ttl that isn't an int now is going to fail, right?

Reply:
Yeah right, it's definetively a BC break. Although I forgot to mention it, I wanted to have your opinion on this. I'll add a function to parse these strings properly to not break backward compatibility :wink:

Reply:
@TerryHowe I fixed it :wink: